packages/braid-design-system/src/lib/components/MenuRenderer/MenuRenderer.snippets.tsx

```diff
@@ -0,0 +1,120 @@
+import type { Snippets } from '../private/Snippets';
+import {
+  MenuRenderer,
+  MenuItem,
+  MenuItemLink,
+  Box,
+  Text,
+  IconChevron,
+  Placeholder,
+  IconList,
+} from '../../playroom/components';
+import source from '@braid-design-system/source.macro';
+import { ButtonIcon } from '../ButtonIcon/ButtonIcon';
+
+export const snippets: Snippets = [
+  {
+    name: 'Text trigger',
+    code: source(
+      <MenuRenderer
+        offsetSpace="small"
+        trigger={(triggerProps, { open }) => (
+          <Box userSelect="none" cursor="pointer" {...triggerProps}>
+            <Text>
+              Menu{' '}
+              <IconChevron
+                direction={open ? 'up' : 'down'}
+                alignY="lowercase"
+              />
+            </Text>
+          </Box>
+        )}
+      >
+        <MenuItem onClick={() => {}}>Button</MenuItem>
+        <MenuItemLink href="#">Link</MenuItemLink>
+      </MenuRenderer>,
+    ),
+  },
+  {
+    name: 'Small Text trigger',
+    code: source(
+      <MenuRenderer
+        size="small"
+        offsetSpace="xsmall"
+        trigger={(triggerProps, { open }) => (
+          <Box userSelect="none" cursor="pointer" {...triggerProps}>
+            <Text size="small">
+              Menu{' '}
+              <IconChevron
+                direction={open ? 'up' : 'down'}
+                alignY="lowercase"
+              />
+            </Text>
+          </Box>
+        )}
+      >
+        <MenuItem onClick={() => {}}>Button</MenuItem>
+        <MenuItemLink href="#">Link</MenuItemLink>
+      </MenuRenderer>,
+    ),
+  },
+  {
+    name: 'ButtonIcon trigger',
+    code: source(
+      <MenuRenderer
+        offsetSpace="small"
+        trigger={(triggerProps) => (
+          <Box userSelect="none" cursor="pointer" {...triggerProps}>
+            <ButtonIcon
+              icon={<IconList />}
+              id="buttonicon-menurenderer"
+              label="Menu"
+            />
+          </Box>
```

Comment:
The `Box` wrapper is not necessary on `ButtonIcon` components, instead `triggerProps` can be spread directly on it.
```suggestion
          <ButtonIcon
            icon={<IconList />}
            id="buttonicon-menurenderer"
            label="Menu"
            {...triggerProps}
          />
```